generator.yaml

```diff
@@ -485,6 +485,48 @@ resources:
         from:
           operation: DescribeMonitoringSchedule
           path: LastMonitoringExecutionSummary
+  FeatureGroup:
+    # CustomUpdateConditions is used for setting terminal
```

Comment:
Do we have comments in the generator.yaml at other places ? I would recommend adding the description to the method instead.